x-pack/plugins/security_solution/public/common/components/sourcerer/utils.tsx

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
FormattedMessage doesn’t accept its props to be variables (The build will fail and complain about this), so I created components to return messages by timeline type.